lib/phoenix_swagger/plug/validate_plug.ex

```diff
@@ -10,19 +10,31 @@ defmodule PhoenixSwagger.Plug.Validate do
   Options:
 
    - `:validation_failed_status` the response status to set when parameter validation fails, defaults to 400.
+   - `:exclude_paths` paths matching these regular expressions will be excluded from validation
   """
   def init(opts), do: opts
 
   def call(conn, opts) do
-    validation_failed_status = Keyword.get(opts, :validation_failed_status, 400)
+    regex_result = opts
+    |> Keyword.get(:exclude_paths, [~r/$^/]) # the default regex matches nothing
```

Comment:
`List.wrap()` will convert a `nil` into an empty list, so no need for the default regex.